cetlib/test/sqlite/connection_t.cc

```diff
@@ -15,7 +16,7 @@
 using namespace std;
 using namespace cet::sqlite;
 using namespace hep::concurrency;
-namespace bfs = boost::filesystem;
+namespace bfs = std::filesystem;
```

Comment:
Retaining `bfs` as a name seems incongruous: could we replace it with `fs`, maybe?